### DISCUSSION TITLE
Supabase Google Auth not working when using the API URL

### DISCUSSION THREAD

**Discussion by fugaku** [supabase/supabase#21355] [Questions] [answered]
navigating to the authorize url e.g. https://[projectid].supabase.co/auth/v1/authorize?provider=google&redirect_to=xxxx
return getaddrinfo ENOTFOUND in postman

![image](https://github.com/supabase/supabase/assets/1614750/62937e72-aee0-4b21-af0d-8971d11012a5)

is this related to ipv4 to ipv6 changes ?

**Comment by GaryAustin1** [comment, marked as answer]
Those changes do not impact the REST API's.  
Something else is going wrong for you.  Was this working and then stopped?
Even if you have the wrong provider names or they are off it returns errors.

Try restarting your instance.

Are other requests to your instance working?
